bin/migrateDirtyDBtoRealDB.js

```diff
@@ -52,6 +50,6 @@ process.on('unhandledRejection', (err) => { throw err; });
   await Promise.all(p);
   console.log(`Wrote all ${numWritten} records`);
 
-  await db.close();
+  await util.promisify(db.close.bind(db))();
```

Comment:
Do you have to restore the `const util = require('util');` line?

Reply:
done